boskos-acquire/0.1/boskos-acquire.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
@bobcatfish I am aware this was submitted before the TEP - ReOrg but since you are adding the annotations and labels, could you also move the task  under the `task` directory so that it shows up in our hub 👼

Reply:
will do, thanks @sthaha !